src/main/java/org/spongepowered/common/mixin/core/entity/item/EntityItemMixin.java

```diff
@@ -192,6 +192,21 @@
             ci.cancel();
         }
     }
+    
+    @Inject(
+        method = "onUpdate", 
+        at = @At (
+            value = "INVOKE", 
+            ordinal = 1, 
```

Comment:
Could this be redone with a slice instead of a single invoke that looks a little vague without looking at the target method?

Reply:
So to be clear, I know nothing about mixins. My implementation was mostly based off of #1917 
and working around until it worked. That being said, I did give this a go trying to read a bit. 
(Honestly mixins aren't something I should be playing with)
I didn't want to upload it, as I have no idea if it is bad practice, but I did test the changes made here, and it still works.

https://gist.github.com/DrZoddiak/408a71583c974392beaf44d235c92c5d